include/kodlab_mjbots_sdk/jointBase.h

```diff
@@ -0,0 +1,93 @@
+
+#ifndef JOINTBASE_H
+#define JOINTBASE_H
+//
+// Created by kodlab on 02/17/22.
+// J. Diego Caporale
+//
+/**
+ * @file jointBase.h
+ * @author Kodlab - J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief Abstract class for joints. Agnostic to servo/motor used.
+ * @version 0.1
+ * @date 2022-02-17
```

Comment:
we should discuss what sort of comments like these we want. My version might not be correct/best